src/main/kotlin/com/hibob/feedback/resource/FeedbackResource.kt

```diff
@@ -53,4 +53,10 @@ class FeedbackResource(private val feedbackService: FeedbackService) {
         }
     }
 
+    @GET
+    @Path("/v1/getFeedbackStatus/{feedbackId}")
+    fun getStatus(activeUser: ActiveUser, @PathParam("feedbackId") feedbackId: UUID): Response {
+        return Response.ok(feedbackService.getStatus(feedbackId, activeUser)).build()
```

Comment:
no permission check?

Reply:
Status Checking API: Provide an API endpoint for employees to check the status of their submitted feedback (if not anonymous).

the permission check verify that you are HR or Admin but I thought that its for the employee only